packages/ui/Touchable/tests/Touchable-specs.js

```diff
@@ -106,6 +132,33 @@ describe('Touchable', () => {
 
 			const ev = {currentTarget: {}};
 			fireEvent.mouseDown(component, ev);
+			fireEvent.mouseEnter(component, ev);
+			rerender(<Component holdConfig={holdConfig} onHold={() => {}} onHoldEnd={handler} />);
+
+			jest.runOnlyPendingTimers();
+
+			fireEvent.mouseUp(component, ev);
+			expect(handler).toHaveBeenCalled();
+			done();
+		});
+
+		test('should update state configurations onHoldEnd events with cancelOnMove=true', (done) => {
```

Comment:
Could you please fix this description?
```suggestion
		test('should update state configurations onHoldEnd events if holdConfig.cancelOnMove is true', (done) => {
```